README.md

```diff
@@ -117,6 +118,17 @@ Sends a request to the remote peer.
     passed by the other peer.
 
 -------------------------------------------------------
+<a name="fire"></a>
+### instance.fire(message, callback(err))
+
+Sends a *fire and forget* request to the remote peer.
+
+  * `message` is a standard JS object, but all streams contained in its
+    `streams` property will be multiplexed and forwarded to the other
+    peer.
+  * `callback` will be called if there is an error while sending the message.
```

Comment:
Mmm, there already is a test that demonstrate the contrary: https://github.com/delvedor/tentacoli/blob/d7f4da799ebda270365e582410c914f5a90f1a04/test.js#L393-L404